rskj-core/src/main/java/co/rsk/rpc/modules/eth/EthModule.java

```diff
@@ -86,7 +87,7 @@ public EthModule(
 
     public Map<String, Object> bridgeState() throws IOException, BlockStoreException {
         Block bestBlock = blockchain.getBestBlock();
-        Repository repository = repositoryLocator.snapshotAt(bestBlock.getHeader()).startTracking();
+        Repository repository = repositoryLocator.startTrackingAt(bestBlock.getHeader());
```

Comment:
I think that through this refactor an `up-to-grab` contribution may have rise. It makes no sense to need the whole repository for obtaining the `bridgeState`, a _reader_ should be safer and enough

Reply:
Yes, I tried but the `Bridge` components both read and write. That's the ISP principle right there.

Reply:
Absolutely, that's why I think it's a pretty safe place for an external contributor to dig in